.github/workflows/check.yml

```
@@ -0,0 +1,90 @@
+name: Check
+on: 
+  pull_request:
+    branches: [ develop ]
+jobs:
+  pr-style:
+    name: pr-style
+    runs-on: ubuntu-latest
+    steps:
+      - uses: deepakputhraya/action-pr-title@master
+        with:
+          regex: '^\[[A-Z]+-\d+\]\s.+' # Regex the title should match.
+  code-style:
+    name: code-style
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install packages
+        run: |
+          sudo apt-get -y install build-essential texi2html
+          
+          # indent 2.2.11, the lastest version (2.2.12) make a different result
+          sudo apt-get -y install wget
+          wget https://ftp.gnu.org/gnu/indent/indent-2.2.11.tar.gz
+          tar xf indent-2.2.11.tar.gz
+          cd indent-2.2.11
+          ./configure
+          sudo make -j install
+          cd ..
+
+          # astyle
+          sudo apt-get -y install astyle
```

```suggestion
        run: |
          # dependencies to build GNU indent
          sudo apt-get -yq install build-essential texi2html wget
          
          # install astyle
          sudo apt-get -yq install astyle
          
          # indent 2.2.11, the lastest version (2.2.12) make a different result
          wget https://ftp.gnu.org/gnu/indent/indent-2.2.11.tar.gz
          tar xf indent-2.2.11.tar.gz
          cd indent-2.2.11
          ./configure
          sudo make -j install
          cd ..
```
1) To suppress output, added q option.
2) move "apt-get install" at the beginning of the script

And could you consider caching apt install by referring to the following?
- https://raymii.org/s/articles/Github_Actions_cpp_boost_cmake_speedup.html (Caching the apt install dependencies section)

---

@hgryoo I'll consider and let you know.